index.html

```diff
@@ -17,11 +17,26 @@
 </head>
 <body>
   <header>
-    <div class="container">
-    </div>
+    <p id="header-p" class="header-P">Code Journal</p>
```

Comment:
Make sure you're utilizing our layout classes! It will handle all of the responsiveness for you essentially. The snippet below is an example
```
<header>
  <div class="container">
    <nav class="navbar column-full">
      <span>Code Journal</span>
    </nav>
  </div>
</header>
  ```